docs/migrationGuide.adoc

```diff
@@ -37,7 +37,7 @@ All remaining brand colors have been removed from DB UI Core and will be used fr
 
 ==== Secondary Colors
 
-All secondary colors as well have been changed to include `db-` at the front, so e.g. `$color-red-500` is now `$db-color-red-500`.
+All secondary colors as well have been changed to include `db-` at the front, so e.g. `$color-red-500` is now `$db-colors-primary-enabled`.
```

Comment:
these are changes to the migrationGuide that is related to Guidelines 2.0 and not Guidelines 3.0. Most likely the change to this file should get reverted in total.

Reply:
fixed